packages/entity-server/src/@types/express/index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import { AccessPolicy } from '@tupaia/access-policy';
+import { Authenticator } from '@tupaia/auth';
+
+declare global {
+  namespace Express {
+    export interface Request {
+      accessPolicy: AccessPolicy;
+      authenticator: Authenticator;
+    }
+  }
+}
```

Comment:
This is how we can use declaration merging to add tupaia specific fields to the express `Request` type